examples/examples.ipynb

```diff
@@ -232,7 +232,7 @@
     "\n",
     "visualizer.fit(X, y)                # Fit the data to the visualizer\n",
     "visualizer.transform(X)             # Transform the data\n",
-    "visualizer.poof()                   # Draw/show/poof the data"
+    "visualizer.show()                   # Draw/show/show the data"
```

Comment:
I will change this one - since it isn't one of the user examples.

Reply:
Actually, the entire examples notebook doesn't work -- @Kautumn06 what's the status of your notebook refactoring?

Reply:
Hi @bbengfort &mdash; I've been working on breaking out the classification and feature visualizers into the smaller notebooks like we talked for Issue #470. However, they are still a work in progress so I doubt I'll have them finished this week. 

I am also open to eliminating the `examples/` directory altogether if you think that makes sense. The text and the code samples in the notebooks are often taken directly from our docs so I think an argument can be made for keeping our focus on updating our documentation rather than trying to maintain two largely interchangeable reference sources.